doc/ale-php.txt

```diff
@@ -336,5 +336,30 @@ g:ale_php_intelephense_config                   *g:ale_php_intelephense_config*
   -docs).
 
 
+===============================================================================
+pint                                                                 ale-pint
+
+g:ale_pint_executable                                   *g:ale_pint_executable*
+g:ale_pint_executable
+Type: |String|
+Default: `'pint'`
+
+This variable sets the executable used for pint.
+
+
+g:ale_pint_options                                         *g:ale_pint_options*
+*b:ale_pint_options*
+Type: |String|
+Default: `''`
+
+This variable can be set to pass additional options to pint.
+
+
+g:ale_pint_use_global                                   *g:ale_pint_use_global*
+*b:ale_pint_use_global*
+Type: |Boolean|
+Default: `get(g:, 'ale_use_global_executables', 0)`
+
+See |ale-integrations-local-executables|
```

Comment:
This whole help section must be after the `php` entry so it matches the same order as in ale.txt file (php -> pint -> tlint).

The right side of the first line must be a vim tag (e.g. surrounded by asteriks \*) and be named ale-php-pint instead of ale-pint:

>   \*ale-php-pint\*


Also all variable names must follow this convention:   ale_php_pint_xxxx.  For example ale_pint_options should be ale_php_pint_options instead and so on.